src/modules/helpchan.ts

```diff
@@ -340,6 +340,8 @@ export class HelpChanModule extends Module {
 		await this.ensureAskChannels(msg.guild!);
 
 		this.busyChannels.delete(claimedChannel.id);
+
+		return msg.channel.send(`👌 successfully claimed ${claimedChannel}`);
```

Comment:
`return` -> `await`
There's no point in returning the message (although it's not bad, cookiecord will just discard it) but `await`-ing is good here as it'll let cookiecord catch the exception that could be made if the bot cannot send the message.

Reply:
I agree there is no point in returning the message, the same command does it in `if (helpUser)` and `if (!claimedChannel)` so I thought I'd do the same here. On `await`ing vs returning the `Promise`, I believe they are both functionally equivalent in this case ([cookiecord `await`s the result](https://github.com/cookiecord/cookiecord/blob/152d8aad837d05a0c88e64fe6f5ec83e178d4dec/src/command/commandParser.ts#L81), so an exception in sending the message would be caught, I'm pretty sure). That being said, I think `await`ing conveys the intent of the code better, I changed it above as well.

Reply:
Ooh, this comment got me confused for a while. I didn't know that you could return a promise in an async function and still have it be a promise. I thought it would return a nested Promise but it seems [not](https://www.typescriptlang.org/play?#code/IYZwngdgxgBAZgV2gFwJYHsIzAU2ACwAoBKGAbwCgYYAnHZBGrABRvQFtUQcA6OkdABsAbjkIBGAAzEA3BQC+FCrgIkZQA).

Reply:
Just looked at the changes to re-review and saw that you actually updated the references other then the last one too. (read last comment for context)

I think `return await` is fine there but it's up to you. Sorry for the confusion.